packages/terra-html-table/src/Table.module.scss

```
@@ -0,0 +1,152 @@
+/* stylelint-disable selector-max-compound-selectors, max-nesting-depth */
+:local {
+  .table {
+    background-color: var(--terra-html-table-background-color, transparent);
+    border: var(--terra-html-table-border, 0);
+    border-collapse: collapse;
+    border-radius: var(--terra-html-table-border-radius, 3px);
+    border-spacing: 0;
+    max-width: 100%;
+    width: 100%;
+
+    caption {
+      padding-bottom: 0.5em;
+    }
+
+    // Header Row, Row
+    tr {
+      display: table-row;
+      margin-bottom: 0;
+    }
+
+    // Row Cell
+    td {
+      border: 0;
+      border-left: var(--terra-html-table-cell-border-left, 0);
+      color: var(--terra-html-table-cell-color, #1c1f21);
+      display: table-cell;
+      font-size: var(--terra-html-table-cell-font-size, 1rem);
+      text-align: left;
+      vertical-align: top;
+
+      &:first-of-type {
+        border-left: var(--terra-html-table-first-cell-border-left, 1px solid #c8cacb);
+      }
+
+      &:last-of-type {
+        border-right: var(--terra-html-table-last-cell-border-right, 1px solid #c8cacb);
+      }
+    }
+
+    // Header Cell
+    th {
+      border: 0;
+      border-left: var(--terra-html-table-header-cell-border-left, 0);
+      display: table-cell;
+      text-align: left;
+      vertical-align: top;
+
+      &:first-of-type {
+        border-left: var(--terra-html-table-first-header-cell-border-left, 1px solid #c8cacb);
+      }
+
+      &:last-of-type {
+        border-right: var(--terra-html-table-last-header-cell-border-right, 1px solid #c8cacb);
+      }
+    }
+
+    // Header
+    thead {
+      background-color: var(--terra-html-table-header-background-color, #dedfe0);
+      box-shadow: var(--terra-html-table-header-box-shadow, 0 4px 4px -4px rgba(134, 138, 140, 0.75), inset 0 0 0 1px #fff);
+      display: table-header-group;
+
+      // Header Row
+      tr {
+        border-bottom: var(--terra-html-table-header-row-border-bottom, 0);
+        border-top: var(--terra-html-table-header-row-border-top, 1px solid #c8cacb);
+      }
+
+      // Header Cell
+      th {
+        color: var(--terra-html-table-header-cell-color, #1c1f21);
+        font-size: var(--terra-html-table-header-cell-font-size, 1rem);
+        font-weight: var(--terra-html-table-header-cell-font-weight, bold);
+        vertical-align: bottom;
+      }
+    }
+
+    // Body
+    tbody {
+      background-color: var(--terra-html-table-body-background-color);
+
+      // Row
+      tr {
+        background-color: var(--terra-html-table-row-background-color, transparent);
+        border-top: var(--terra-html-table-row-border-top, 1px solid #c8cacb);
```

Plus add new `&:first-of-type` to not collapse/override over the `thead tr` border-bottom above

```suggestion
        border-top: var(--terra-html-table-row-border-top, 1px solid #dedfe0);
        
        &:first-of-type {
          border-top: 0;
        }
```